src/Components/Projects/Projects.tsx

```diff
@@ -153,10 +153,9 @@ const Projects: React.FC = () => {
   };
   // **********search by proj name***********************
   const getProjectTitleValue = (e: MouseEvent) => {
-    console.log("search", e.target.value);
```

Comment:
you can also comment the console.logs, instead of completely removing them, as you might need them later